### ISSUE TITLE
[TF2] Players with Ping above 100 cause the number to not fit on the Scoreboard

### ISSUE THREAD

**Issue by gaelcoral** [ValveSoftware/Source-1-Games#6519]
I'm surprised no one has reported this before, but when a player has a ping above 100, the number doesn’t fit correctly on the scoreboard
 
 
![20241113074317_1](https://github.com/user-attachments/assets/9413c8c6-1707-4170-b42b-6652ba3ffcbf)

**Comment by jh34ghu43gu** [comment]
Probably because it's not an issue for others (me)
![image](https://github.com/user-attachments/assets/5649a49f-6698-45d2-b9cb-1774ed19ac90)
1440p resolution with `vgui_ui_scale_factor 1` `vgui_ui_scale_width 2560` and `vgui_ui_scale_height 1440`
No custom hud

**Comment by gaelcoral** [comment]
> Probably because it's not an issue for others (me)
1440p resolution with vgui_ui_scale_factor 1 vgui_ui_scale_width 2560 and vgui_ui_scale_height 1440
No custom hud

Well, I tried it in alt account in Geforce Now, with the default settings in 1080P and the same issue as above
![image](https://github.com/user-attachments/assets/37a60486-1d53-43e0-8c36-8165e1caa09c)

**Comment by CriticalFlaw** [comment]
In `resource/ui/scoreboard.res` increasing the value of `ping_width` from 15 to 20 should do the trick. If you have a bot in the server, the BOT text will also be cutoff with how things are today.

**Comment by gdfsgdfg** [comment]
yep scoreboard is still messed up for me on 1080p cutting off ping numbers and bot
